src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -242,9 +262,44 @@ public boolean updateTeam(Id teamId, Team updatedTeam) {
      * @throws AlfredException
      */
     public void addTeam(Team team) throws AlfredException {
+        // TODO: Verify that participants and mentors are in their respective list before adding.
         this.teamList.add(team);
     }
 
+    /**
+     * Adds the participant to the given team.
+     *
+     * @param teamId
+     * @param participant
+     * @throws AlfredException if the team does not exist.
+     */
+    public void addParticipantToTeam(Id teamId, Participant participant) throws AlfredException {
```

Comment:
Will the participant be a clone or does it not matter?

Reply:
Do you mean clone as in shares the same fields as another object but has a different pointer address? If so, yea it does not matter because we overrode equals method